config/default/system.theme.yml

```diff
@@ -1,4 +1,4 @@
 admin: claro
-default: uiowa_bootstrap
+default: uids_base
```

Comment:
We will probably not want to make this update quite yet, unless there is no way to roll this out until we update all sites to use uids_base.

Reply:
Agreed!